Improvements to the portal

scrnetto opened the pull request:
Add a basic system for "verified coin profiles", e.g; social links, website, etc.
Add 'open full image' button / clickable image to the NFT Details view.
Replace "Burned NFTs" with total sales on the SC Marketplace.
Add favicon
Replace SCPscan with logo in Navbar
Show table with all downloads below the auto detection
Move Dark/Light mode switch button to right in navbar

Commit:
Add 'open full image' button / clickable image to the NFT Details view.

Replace "Burned NFTs" with another stat in the NFTs list of collections.

Add favicon

Replace SCPscan with logo in Navbar

Show table with all downloads below the auto detection

Move Dark/Light mode switch button to right in navbar

Commit:
Add a basic system for "verified coin profiles"

sc-9310 commented on public/js/nfts.js:
```diff
@@ -56,11 +57,34 @@ function renderNFTs(query = '') {
+			var nftsForSale = res.result.nftsForSale.length;
+			
+			if (nftsSold > 0){
+				domTotalBurnsTxt.innerText = 'Total sold in the Marketplace';
```
Why are the variables not re-named?

scrnetto replied:
If you are referring to domTotalBurnsTxt, the name is always the same because the id of the container div always remains the same

sc-9310 commented on public/js/nfts.js:
```diff
@@ -56,11 +57,34 @@ function renderNFTs(query = '') {
+});
+
+function getInfoCollection(){
+	$.getJSON('https://stakecube.io/api/v2/marketplace/collection?contract=' + id, function(res) {
```
Tbh, im not sure about this one in general. I would vote **against** pulling stats from a _centralized third party_. This makes SCPscan depending on others (even its from the same Devs) and one advantage is, SCPscan can run standalone

cc @JSKitty

scrnetto replied:
We can find other values to display if there are no NFTs burned in the collection. However, if the marketplace API were no longer functional, those values would not be entered.

scrnetto replied:
We can find other values to display if there are no NFTs burned in the collection. However, if the marketplace API were no longer functional, those values would not be entered.

JSKitty replied:
I think it's fine to display some centralized info if it is non-mandatory data, like the Verification data is pulled for the Collections ranking, but if the request fails, the site will just fallback to other data, it won't break, a similar method here would be awesome.

JSKitty commented on public/collections.html:
```diff
@@ -25,14 +25,16 @@
   <script src="js/theme.js"></script>
   <script src="js/utils.js"></script>
+  
+  <link rel="shortcut icon" type="image/ico" href="img/scp.ico" />
```
The favicon (.ico) is visibly "squashed" in the browser tab, as shown below (top left + opened ico file):
![image](https://user-images.githubusercontent.com/42538664/149026584-097ff55f-a310-404a-9522-09170e988c09.png)

This is due to the icon file being 'cut down' to fit the logo (which is indeed higher than the width), but this can be easily fixed by adding some width resolution to the image.

An easy alternative is to just use the .ico file from the SCP Wallet repository, as the ICO files there are perfectly aligned and sized.

JSKitty reviewed (changes requested):
I left an in-line code review, and have two more reviews below for bugs that I don't know the origin of:

In the Download page: Sometimes executables will be tagged as "signatures", the v1.1.7 release binary was tagged as a signature, but the v1.1.6 release was tagged correctly.
![image](https://user-images.githubusercontent.com/42538664/149028360-9ce7c2b7-e55a-4fb2-bd8f-b21362e9e3b7.png)

And in the home page (Tokens), clicking on "Total Users" will popup a very dark overlay, and then the page becomes unusable: I cannot click anything without refreshing the page, seems to be some CSS issue caused by this PR.

Other than these issues, the PR seems great: the download page is awesome, logo looks nice on all pages, etc.

Commit:
Corrections required

Changed image ico
Fixed modal problem
Fixed the recognition of the file type on the downloads page

scrnetto commented:
@JSKitty I have made the required corrections

JSKitty commented:
Perfect! All issues were fully corrected, I've tested all. 😄
